docs/atlascli/command/atlas-deployments-connect.txt

```diff
@@ -0,0 +1,78 @@
+.. _atlas-deployments-connect:
+
+=========================
+atlas deployments connect
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Connects to a deployment.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas deployments connect [deploymentName] [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - deploymentName
+     - string
+     - false
+     - Name of the deployment you want to connect.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --connectWith
+     - string
+     - false
+     - Method of connection. Valid values are mongosh, compass and skip.
+   * - -h, --help
+     - 
+     - false
+     - help for connect
```

Comment:
```suggestion
     - Help for connect.
```

Reply:
the --help flag (and the documentation) is not controlled by us, but by the [cobra lib](https://github.com/spf13/cobra/blob/0c72800b8dba637092b57a955ecee75949e79a73/command.go#L1164).
For all existing cli commands the --help is documented using the same format "help for <cmd>".
Example: https://www.mongodb.com/docs/atlas/cli/stable/command/atlas-accessLists-list/